.github/workflows/lintcommit.yml

```diff
@@ -4,6 +4,7 @@ on:
     types: [opened, synchronize, reopened, ready_for_review]
     branches:
       - 'master'
+      - 'release-[0-9]+.[0-9]+'
```

Comment:
do we have (or plan to have) `release-x.y` branches?

Reply:
Huh, good catch. I just copy pasted it from other workflows but all of them are wrong lol.

Reply:
Wait no, this is right though. We do have branches of that form: `release-0.5` would be `x=0` and `y=5`.

Reply:
Ah, didn't think that far ahead. Smurt.